lib/themes/dosomething/paraneue_dosomething/js/components/onboard.js

```diff
@@ -0,0 +1,21 @@
+const React = require('react');
+const ReactDOM = require('react-dom');
+
+import Carousel from './carousel.js';
+
+const Onboard = React.createClass({
+  render: function() {
+    return (
+      <div className="onboard">
+        <Carousel />
+      </div>
+    );
+  }
+});
+
+if (localStorage.getItem('dosomething_enable_onboarding') == "true") {
```

Comment:
connivence of not having to touch drupal haha, but if we're going to make drupal logic now anyway ill just switch this for a feature flag